grails-app/controllers/org/pih/warehouse/api/ProductApiController.groovy

```diff
@@ -287,4 +289,12 @@ class ProductApiController extends BaseDomainApiController {
 
         render([data: tags] as JSON)
     }
+
+    def glAccountOptions = {
+        def glAccounts = GlAccount.list().collect {
+            [id: it.id, label: "${it.code} - ${it.name}"]
+        }
+
+        render([data: glAccounts] as JSON)
+    }
```

Comment:
this should be in the `GlAccountController`, not in the `ProductApiController` and would be good to add this fetching stuff to the service and here leave:
```
def glAccounts = glAccountService.getGlAccountsOptions()
...
```